action.yml

```diff
@@ -33,6 +28,9 @@ inputs:
     description: 'Enable reporting of schema insights (requires ariga-token)'
     default: 'true'
     required: false
+  project-env:
+    description: "The env from the atlas.hcl project file to use"
```

Comment:
I think `The environment name` is enough, no need to couple it with where it came from